documentation/decisions/adr/0004-db-planning.md

```diff
@@ -0,0 +1,63 @@
+# DB Planning
+
+- **Status:** WIP
```

Comment:
```suggestion
- **Status:** Approved
```

Reply:
We just discussed that we can make the status `Approved` even as we are drafting so that it doesn't get forgotten when merging.